vendor/github.com/DataDog/datadog-go/statsd/statsd.go

```diff
@@ -320,7 +361,7 @@ func (c *Client) sendMsg(msg string) error {
 // send handles sampling and sends the message over UDP. It also adds global namespace prefixes and tags.
 func (c *Client) send(name string, value interface{}, suffix []byte, tags []string, rate float64) error {
 	if c == nil {
-		return nil
+		return fmt.Errorf("Client is nil")
```

Comment:
The test failure seems to be related to this change. We weren't expecting a `nil` statsd client to raise, but maybe we ought to? I'm not sure.

Reply:
Made DataDog/datadog-go#90 to fix this. Once it's merged & pulled in, we'll have to adjust the test case to check for the error constant.

Reply:
Ah, thanks should have run locally before opening. Also TIL, `go test  ./...` to run all tests.
Do you want me to change the constraint in gopkg.toml to specific sha and fix tests or should we actually wait for new release version of datadog client.

Reply:
I think it should be fine to update to the git version, especially if the bug fixed you are pulling in are blocking you!

Note that the test will need an update also, to validate that `ErrNoClient` is returned in the case of an empty client.